whats_fresh/whats_fresh_api/templates/story.html

```diff
@@ -67,24 +67,39 @@ <h4 id='error_tag' class='light warning'></h4>
         {% else %}
         <span class="no_image">No Images Yet!</span>
         {% endif %}
+
     </div>
 
-        <div class="video_container">
-        <h3>Videos</h3>
-        <button type="button" onclick="appendVideo()">Add Video</button>
-        {% if existing_videos %}
-        {% for video in existing_videos %}
-        <div class="video">
-            <select disabled id="existing{{ prep.id }}" class="video_select">
-                <option value="{{ video.id }}">{{ video.name }}</option>
-            </select>
-            <a href="#" onclick="deleteVideo(this);return false;"><img class="delete" src="{% static 'delete.png' %}" /></a>
-        </div>
-        {% endfor %}
-        {% else %}
-        <span class="no_video">No Videos Yet!</span>
-        {% endif %}
+    <div id="inline_image">  <!-- Div used if a new inline image submitted -->
+    </div>
+
+    <button type="submit"  onclick="return imagePopup('/entry/stories/new/images/new/')">Add New Images</button>
+    <div id="image-popup"></div>
+
+    <div class="video_container">
+    <h3>Videos</h3>
+    <button type="button" onclick="appendVideo()">Add Video</button>
+    {% if existing_videos %}
+    {% for video in existing_videos %}
+    <div class="video">
+        <select disabled id="existing{{ prep.id }}" class="video_select">
+            <option value="{{ video.id }}">{{ video.name }}</option>
+        </select>
+        <a href="#" onclick="deleteVideo(this);return false;"><img class="delete" src="{% static 'delete.png' %}" /></a>
     </div>
+    {% endfor %}
+    {% else %}
+    <span class="no_video">No Videos Yet!</span>
+    {% endif %}
+
+    </div>
+
+    <div id="inline_video">  <!-- Div used if a new inline video submitted -->
+    </div>
+
+    <button type="submit" id="newImage-btn" onclick="return videoPopup('/entry/stories/new/videos/new/')">Add New Videos</button>
```

Comment:
This is on the video button.

Also, can it be Add New Video instead of Add New Videos?